gulpfile.js

```diff
@@ -5,6 +5,7 @@ requireDir('./config/gulp');
 
 gulp.task('default', [
     'twig',
+    'favicon',
```

Comment:
remove favicon task from default task, if we wanna run this task, we will write 'gulp favicon' seperate in the terminal

Reply:
Removed...